GoogleUtilities/Example/Tests/Swizzler/GULAppDelegateSwizzlerTest.m

```diff
@@ -667,72 +674,76 @@ - (void)testAppDelegateInstance {
 #if TARGET_OS_IOS || TARGET_OS_TV
 /** Tests that application:openURL:options: is invoked on the interceptor if it exists. */
 - (void)testApplicationOpenURLOptionsIsInvokedOnInterceptors {
-  id interceptor = OCMProtocolMock(@protocol(GULApplicationDelegate));
-  OCMExpect([interceptor application:OCMOCK_ANY openURL:OCMOCK_ANY options:OCMOCK_ANY])
-      .andReturn(NO);
-
-  id interceptor2 = OCMProtocolMock(@protocol(GULApplicationDelegate));
-  OCMExpect([interceptor2 application:OCMOCK_ANY openURL:OCMOCK_ANY options:OCMOCK_ANY])
-      .andReturn(NO);
-
-  NSURL *testURL = [[NSURL alloc] initWithString:@"https://www.google.com"];
-  NSDictionary *testOpenURLOptions = @{UIApplicationOpenURLOptionUniversalLinksOnly : @"test"};
-
-  GULTestAppDelegate *testAppDelegate = [[GULTestAppDelegate alloc] init];
-  OCMStub([self.mockSharedApplication delegate]).andReturn(testAppDelegate);
-
-  [GULAppDelegateSwizzler proxyOriginalDelegate];
-  [GULAppDelegateSwizzler registerAppDelegateInterceptor:interceptor];
-  [GULAppDelegateSwizzler registerAppDelegateInterceptor:interceptor2];
-
-  [testAppDelegate application:[GULApplication sharedApplication]
-                       openURL:testURL
-                       options:testOpenURLOptions];
-  OCMVerifyAll(interceptor);
-  OCMVerifyAll(interceptor2);
-
-  // Check that original implementation was called with proper parameters
-  XCTAssertEqual(testAppDelegate.application, [GULApplication sharedApplication]);
-  XCTAssertEqual(testAppDelegate.url, testURL);
+  if (@available(iOS 10, *)) {
```

Comment:
Should there be alternate tests for iOS 9 and 10?

Reply:
Sure, there is one, see tests around `[UIApplication application:openURL:sourceApplication:annotation:]`